package.json

```diff
@@ -7,8 +7,8 @@
     "build": "ncc build src/index.ts",
     "lint": "eslint . --ext .ts,.tsx",
     "format": "prettier --write **/*.ts",
-    "test": "mocha -r ts-node/register -t 120s 'tests/unit/*.test.ts'",
-    "e2e-tests": "mocha -r ts-node/register -t 120s 'tests/e2e.test.ts'"
+    "test": "mocha -r ts-node/register -t 120s 'tests/unit/*.test.ts' --exit",
```

Comment:
Is there a reason why mocha wont quit?

Reply:
I find that the tests do hang without this exit. There may be something that the tests are waiting to be manually closed but this is easier and common